lib/capture_go_web/templates/games/live_game.html.leex

```diff
@@ -1,7 +1,21 @@
 <h1><%= @game.name %> (Game #<%= @game.id %>)</h1>
 
-<svg style="width:1000px;height:1000px;">
-  <%= nine_by_nine() %>
-  <%= star_points() %>
-  <%= render_stones(@game.goban) %>
-</svg>
+<div class="columns">
+
+<div class="column">
+  <svg style="width:1000px;height:1000px;">
+    <%= nine_by_nine() %>
+    <%= star_points() %>
+    <%= render_stones(@game.goban) %>
+  </svg>
+</div>
+
+<div class="column">
+  <%= if participant?(@game, @current_user) do %>
+    <button class="button" phx-click="pass">
+      Pass turn
```

Comment:
This should be grayed out or something when it's not your turn.
We also need some turn indicator in general.

Reply:
https://github.com/Giesch/capture_go/pull/1/commits/ddfa7074a5ec2ac6fc9444cb7d183632069c1614